pkg/kubectl/cmd/create.go

```diff
@@ -69,7 +71,7 @@ func ValidateArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func RunCreate(f *cmdutil.Factory, out io.Writer, filenames util.StringList) error {
+func RunCreate(f *cmdutil.Factory, out io.Writer, filenames util.StringList, cmd *cobra.Command) error {
```

Comment:
I think it was deliberate to not pass the cobra command to these functions. See, for instance, #4470